app/controllers/messages_controller.rb

```diff
@@ -2,8 +2,13 @@ class MessagesController < ApplicationController
 	before_action :authenticate_user!
 	before_action :authenticate_control!
 	#Displays all messages, newest ones first
+
+	layout "mobile"
+
 	def index
 		@messages = Message.all.order('created_at DESC')
+		@newMessage = Message.new
```

Comment:
The index will likely need to change so that I'm only getting messages that are "mine" (either ones I've sent or ones sent to me), but right now I didn't do that. 

What I _did_ do is add the Message.new parameter to be passed into the index. This allows us to have a form and hit the "create" action without actually having to change pages. Which means that we're not actually using the `Show` action on this controller at all.